src/main/java/seedu/address/ui/TutorListPanel.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.Student;
+
+/**
+ * Panel containing the list of tutors.
+ */
+public class TutorListPanel extends UiPart<Region> {
+    private static final String FXML = "TutorListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(TutorListPanel.class);
+
+    @javafx.fxml.FXML
+    private ListView<Student> tutorListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
+     */
+    public TutorListPanel(ObservableList<Student> tutorList) {
+        super(FXML);
+        tutorListView.setItems(tutorList);
+        tutorListView.setCellFactory(listView -> new TutorListPanel.TutorListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
```

Comment:
Should this be Student and TutorCard instead?

Reply:
Fixed, thanks! 👍